srcpkgs/qt5pas/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'qt5pas'
 pkgname=qt5pas
-version=2.6~beta
+version=2.2.0
```

Comment:
I am pretty sure this will be considered a downgrade by xbps, so people won't get it

Reply:
I don't understand that much, but the current version is more outdated, I didn't understand why they put 2.6 beta when it wasn't like that